js/src/constants.js

```diff
@@ -123,3 +123,12 @@ export const GOOGLE_WPCOM_APP_CONNECTED_STATUS = {
 	ERROR: 'error',
 	DISABLED: 'disabled',
 };
+
+export const NON_INTERACTABLE_PROPS = {
```

Comment:
```suggestion
export const NON_INTERACTABLE_SELECT_PROPS = {
```
@ankitrox Let's rename the constant to `NON_INTERACTABLE_SELECT_PROPS` because `NON_INTERACTABLE_PROPS` is too generic.